CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
-# Change Log
+## Change Log
```

Comment:
Why change the headings? Normally they should start at level 1.

Reply:
Well, it's used this way at nREPL CHANGELOG, but I'll keep this starting from level 1 then o/

Reply:
Ah, good point. It wasn't created by me and I guess I didn't pay attention to this small inconsistency. Probably we should fix it instead. :)